shared/src/gas_price_estimation.rs

```diff
@@ -57,7 +60,14 @@ pub async fn create_priority_estimator(
                 if !is_mainnet(&network_id) {
                     return Err(anyhow!("GasNow only supports mainnet"));
                 }
-                estimators.push(Box::new(GasNowGasStation::new(client.clone())))
+                let mut estimator = GasNowWebSocketGasStation::new(Duration::from_secs(30));
```

Comment:
ubernit: Can this value be a named constant just so I know what the 30 seconds are? In theory I could just look it up in the `gas-esimation` crate what it means? Something like:
```suggestion
                let max_update_age = Duration::from_secs(30);
                let mut estimator = GasNowWebSocketGasStation::new(max_update_age);
```

No strong feelings though, so feel free to ignore.